Fixed #35130 -- Doc'd django.db.close_old_connections(). (#17766)
django/django: docs/ref/databases.txt

If a connection is created in a long-running process, outside of Django’s
request-response cycle, the connection will remain open until explicitly

> closed, or timeout occurs. In order to close the connection use 
> ``django.db.close_old_connections()``.

Reviewer: Maybe:
```suggestion
closed, or timeout occurs. You can use ``django.db.close_old_connections()`` to
close all old or unusable connections.
```

I think we also should add it to the `django.db.__init__.__all__`.
